assertj-core/src/test/java/org/assertj/core/error/ShouldContainSubsequenceOfCharSequence_create_Test.java

```diff
@@ -57,10 +59,28 @@ void should_create_error_message_with_custom_comparison_strategy() {
     // THEN
     then(message).isEqualTo(format("[Test] %nExpecting actual:%n" +
                                    "  \"" + actual + "\"%n" +
-                                   "to contain the following CharSequences in this order:%n" +
+                                   "to contain the following CharSequences in this order (possibly with other values between them):%n" +
                                    "  [\"{\", \"author\", \"title\", \"}\"]%n" +
                                    "but \"title\" was found before \"author\"%n" +
                                    "when comparing values using CaseInsensitiveStringComparator"));
   }
 
+  @Test
+  void should_create_error_message_indicating_duplicate_subsequence() {
+    // GIVEN
+    String[] sequenceValues = { "{", "title", "author", "title", "}" };
+    String actual = "{ 'title':'A Game of Thrones', 'author':'George Martin'}";
+    ErrorMessageFactory factory = shouldContainSubsequence(actual, sequenceValues, Map.of("title", 1),
```

Comment:
As the language level is 8, you could use instead `org.assertj.core.test.Maps.mapOf` and `org.assertj.core.api.Assertions.entry`:

```suggestion
    ErrorMessageFactory factory = shouldContainSubsequence(actual, sequenceValues, mapOf(entry("title", 1)),
```